src/guiFormSpecMenu.cpp

```diff
@@ -3793,3 +3927,19 @@ std::wstring GUIFormSpecMenu::getLabelByID(s32 id)
 	}
 	return L"";
 }
+
+/**
+ * get ftype of element by element id
+ * @param id of element
+ * @return FormSpecFieldType or f_Unknown
+ */
+FormspecFieldType GUIFormSpecMenu::getTypeByID(s32 id)
+{
+	for(std::vector<FieldSpec>::iterator iter =  m_fields.begin();
```

Comment:
The other spots i got, but this one i don't quite understand. What's wrong with the spaces here? Basically made it look like the two similar functions above it.